src/Markdig/Parsers/FencedBlockParserBase.cs

```diff
@@ -45,7 +44,7 @@ public abstract class FencedBlockParserBase<T> : FencedBlockParserBase where T :
         /// </summary>
         protected FencedBlockParserBase()
         {
-            InfoParser = DefaultInfoParser;
```

Comment:
Shouldn't the default parser reroute internally to the previous default or round trip info parser depending on whether the `BlockProcessor`.`TrackTrivia` ?
Don't think that we want separated default + roundtripinfo if things are triggered by the presence of TrackTrivia

Reply:
fixed, thanks for spotting. This should improve perf slightly.